omicron/cli/process.py

```diff
@@ -540,12 +545,12 @@ def main(args=None):
     logger.debug('Omicron version: %s' % omicronv)
 
     # -- parse configuration file and get parameters --------------------------
-    config_path = Path(args.config_file)
-    if not config_path.is_file():
-        logger.critical(f'Configuration file : {str(config_path.absolute())} does not exsit')
-
+    config_file = Path(args.config_file)
+    if not config_file.exists:
+        logger.critical(f'configuration file does not exist: {config_file.absolute()}')
+        exit(4)
```

Comment:
Why is this using `sys.exit` and not raising an exception?

Reply:
OK, how about FileNotFoundError wit the same text?